ChangeLog.d/bugfix_PR3294.txt

```diff
@@ -0,0 +1,8 @@
+Bugfix
+   * Add guards in pk_wrap.c to ensure if ECDSA is not defined, errors are
```

Comment:
Suggestion: Focus on the _Why?_ instead of the _What?_: E.g. "Fix build failure in configurations where PSA Crypto is enabled but ECDSA is disabled."